BUILDING.md

```diff
@@ -68,28 +68,34 @@ Docker's [post-installation steps for Linux](https://docs.docker.com/install/lin
 
 To build an image, run:
 
-```
+```shell
 cargo make
 ```
 
-This will build an image for the default variant, `aws-k8s-1.21`.
+This will build an image for the default variant (a recent `aws-k8s-*`, see [Makefile.toml](Makefile.toml)).
```

Comment:
Should we just point to the `BUILDSYS_VARIANT` line here?

https://github.com/bottlerocket-os/bottlerocket/blob/ac833d8191a235bd85c185bb30c22647ea9d7013/Makefile.toml#L21

Reply:
> Should we just point to the BUILDSYS_VARIANT line here?

I think that would be brittle, but we could augment the sentence, something like:

```suggestion
This will build an image for the default variant (a recent `aws-k8s-*`, see the `BUILDSYS_VARIANT` variable in [Makefile.toml](Makefile.toml) to find the current default variant).
```

Reply:
Made this change. 👍